<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 0, 130);</string>
  </property>
  <widget class="QPushButton" name="LogupButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>140</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 12, 255);font-size:19pt;color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Don't have an account?</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="createaccbutton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 12, 255);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Создать акаунт</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>311</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>15</number>
    </property>
    <property name="verticalSpacing">
     <number>8</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font-size:15pt; color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="password">
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="username">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:15pt; color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Логин</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="username_2">
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="password_2">
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font-size:15pt; color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Вывод</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>60</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); font-size:28pt;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
